packages/react-scripts/scripts/build.js

```diff
@@ -56,6 +57,10 @@ if (!checkRequiredFiles([paths.appHtml, paths.appIndexJs])) {
   process.exit(1);
 }
 
+// Process CLI arguments
+const argv = process.argv.slice(2);
+const writeStatsJson = argv.indexOf('--stats') !== -1;
```

Comment:
Not sure if `stats` is the best name... Open to suggestions!

Reply:
I think `--stats` is quite nice, as it pairs well with `webpack --stats` flag.